OMPython/__init__.py

```diff
@@ -238,8 +239,11 @@ def sendExpression(self, command):
             self._omc = None
             return result
           else:
-            answer = OMTypedParser.parseString(result)
-            return answer
+            try:
+              answer = OMTypedParser.parseString(result)
+              return answer
+            except:
```

Comment:
No, returning `result` is not fine. How do you distinguish between a function returning a string and a function giving an error? If you wanted just the result, just do a call that does not expect the function to be possible to parse. Or add another OMPython call that tries to parse the string and otherwise always returns the result.